pkg/oci/oci.go

```diff
@@ -0,0 +1,53 @@
+package oci
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+
+	"oras.land/oras-go/pkg/auth"
+	dockerauth "oras.land/oras-go/pkg/auth/docker"
+	"oras.land/oras-go/pkg/content"
+	"oras.land/oras-go/pkg/oras"
+)
+
+const tempFilePrefix = "gator-bundle-"
```

Comment:
(thanks for moving this to its own package 💯 )

could we actually parametrize this  into the function calls now since it's not a `gator` only thing anymore?

Reply:
What sense is there in having a non-constant (user configurable?) file prefix?

Reply:
I don't think the prefix needs to be configured. One thought: are the temporary directories ever GC'd?

Reply:
yep, the directories are deleted after they've been read into memory